tests/test_freqtradebot.py

```diff
@@ -743,6 +743,7 @@ def test_execute_entry(mocker, default_conf_usdt, fee, limit_order,
         (open_rate - (wallet_balance / position)) / (1 - (mm_ratio + taker_fee_rate))
         (10 - (2 / 1)) / (1 - (0.01 + 0.0006)) = 8.085708510208207
     """
+    # TODO: Parametrize this test
```

Comment:
This test is parametrized.
parametrizing it further will only obscure what it's actually testing.

if any, it should be split up into multiple tests.